content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_clusters.rb

```diff
@@ -0,0 +1,45 @@
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Transform
+          module Import
+            class ListClusters
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                values_hash = {}
+                values_hash[''] = '-- select cluster from list --'
+
+                provider_id = @handle.root['dialog_provider']
```

Comment:
@mkanoor The `'!'` value of `provider_id` is being handled at line 17, if that is your question.